src/index.js

```diff
@@ -5,18 +5,35 @@
  *  @description
  *  The CARTO VL functionality is exposed through the **carto** namespace including:
  *
+ * Current version:
  * - {@link carto.version|carto.version}
+ *
+ * Setup:
+ * - {@link carto.setDefaultAuth|carto.setDefaultAuth}
+ * - {@link carto.setDefaultConfig|carto.setDefaultConfig}
+ *
+ * CARTO Basemaps:
+ * - {@link carto.basemaps.voyager|carto.basemaps.voyager}
+ * - {@link carto.basemaps.darkmatter|carto.basemaps.darkmatter}
+ * - {@link carto.basemaps.positron|carto.basemaps.positron}
+ *
+ * Source:
  * - {@link carto.source.Dataset|carto.source.Dataset}
- * - {@link carto.source.SQL|carto.source.SQL}
  * - {@link carto.source.GeoJSON|carto.source.GeoJSON}
  * - {@link carto.source.MVT|carto.source.MVT}
- * - {@link carto.source.MVT.Metadata|carto.source.MVT.Metadata}
- * - {@link carto.expressions|carto.expressions}
+ * - {@link carto.source.SQL|carto.source.SQL}
+ *
+ * Layer:
  * - {@link carto.Layer|carto.Layer}
+ *
+ * Viz:
  * - {@link carto.Viz|carto.Viz}
+ *
+ * Expressions:
+ * - {@link carto.expressions|carto.expressions}
+ *
+ * Interactivity:
  * - {@link carto.Interactivity|carto.Interactivity}
- * - {@link carto.setDefaultAuth|carto.setDefaultAuth}
- * - {@link carto.setDefaultConfig|carto.setDefaultConfig}
  */
```

Comment:
`on` & `off` functions are missing. They used to be under the old "Functions" header (with setDefaultAuth / setDefaultConfig). Maybe they could appear under a new one called "Events".

Reply:
Hmmm yes, I have to take a look to this. Thanks!

Reply:
After taking a look, they are defined in the interactivity section.

Reply:
Ok, but we have to be sure that is not misleading, cause the are different funcions: carto.on / carto.off & interactivity.on / interactivity.off